### ISSUE TITLE
OSRS Touchbar for Macbook's

### ISSUE THREAD

**Issue by james6213** [runelite/runelite#12080]
So, I play Runescape on my PC but also on my Macbook for when im laying in bed, the one annoying thing is for the touchbar not displaying any FKeys it's pretty frustrating as it rules out doing bossing/anything requiring switches for me anyways as OSRS has turned into a game where bosses require a lot of switching to happen which is great, just sucks when your on a Macbook with a touchbar!

I did some research and found someone did create a touch bar for OSRS but it doesn't function very well as I would have to load up a seperate application everytime I load up Runelite! I don't believe this feature would be incredibly hard to develop into Runelite especially if you was to work with the code already developed (I might be wrong, things always do seem a hell of a lot easier when you don't know how much work is really needed for a feature!) 

The current application I use is from the following GIT Repo - https://github.com/pg8wood/OSRS-touch-bar

Basically giving functionality to use the touchbar and move certain keys around on the touchbar, would be awesome if there was a plugin like this on Runelite, and yes I know this is probably not the most sought after development from you guys, but I think a lot of people would benefit and appreciate an update like this, I could be wrong but I think it'd be awesome!

Please let me know what you guys think of this, and I completely understand if this isn't something that you guys will be developing on, just feel like it would be a proper sweet update to a lot of players!!

**Comment by melkypie** [comment]
Duplicate of #11584

**Comment by Broooklyn** [comment]
Just an FYI, you can set it to always display fkeys when you're focused on RuneLite:
https://www.imore.com/how-make-function-keys-default-touch-bar-display

Alternatively, enable the `Key Remapping` plugin and you can use number keys as fkeys.
